<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputOzoneForm</class>
 <widget class="QWidget" name="OutputOzoneForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>277</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>277</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>277</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Outputting Ozone</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="diagRight">
   <property name="geometry">
    <rect>
     <x>422</x>
     <y>217</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="diagLeft">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="DiagnosticLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Diagnostics</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="OutputLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>451</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>O3 Output: 0 ppb</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="FlowLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TempIconLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>B</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LightIconLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>C</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>93</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QWidget" name="widgetBackground" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="BlockingLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/Resources/hourGlassPleaseWait.png</pixmap>
   </property>
  </widget>
  <zorder>widgetBackground</zorder>
  <zorder>Title</zorder>
  <zorder>diagRight</zorder>
  <zorder>diagLeft</zorder>
  <zorder>DiagnosticLabel</zorder>
  <zorder>OutputLabel</zorder>
  <zorder>FlowLabel</zorder>
  <zorder>TempIconLabel</zorder>
  <zorder>LightIconLabel</zorder>
  <zorder>quitButton</zorder>
  <zorder>BlockingLabel</zorder>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
